ddtrace/appsec/_asm_request_context.py

```diff
@@ -94,6 +108,12 @@ def set_value(category, address, value):  # type: (str, str, Any) -> None
         asm_context_attr[address] = value
 
 
+def set_waf_address(address, value, span):  # type: (str, Any, Any) -> None
+    set_value("waf_addresses", address, value)
```

Comment:
minor: use a private constant for the object attribute (category)

Reply:
Yes, this is done for all string literals now.